_shared/project/conf/supervisord-dev.conf

```diff
@@ -0,0 +1,43 @@
+[supervisord]
```

Comment:
This is mostly the same `supervisord-dev.conf` file as from `pyramid-app` on `main`, with just one small change.